src/ucs/debug/instrument.h

```diff
@@ -88,20 +108,29 @@ static inline unsigned long ucs_instrument_expand_location(uint32_t location,
  */
 extern ucs_instrument_context_t ucs_instr_ctx;
 
+uint32_t __ucs_instrument_register(ucs_instrumentation_types_t type,
```

Comment:
please document the possible return values of this function and their meaning (e.g 0 - disabled)